sensor_msgs/Image image
---
float32[] global_desc
geometry_msgs/Point keypoints

#Every local_descriptor is 256 dimenision
float32[] local_descriptors